src/components/atoms/ButtonGroupPicker/ButtonGroupPicker.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// C O M P O N E N T
+export const ButtonGroupPicker: React.FC = (props) => {
+  const { children } = props
+  return (
+    <StyledButtonGroupPicker className='button-group-picker'>
+      {children}
+    </StyledButtonGroupPicker>
+  )
+}
+
+// S T Y L E S
+const StyledButtonGroupPicker = styled.div`
+  display: flex;
+
+  > .btn {
+    position: relative;
+    border-radius: 0;
+    border-radius: none;
```

Comment:
no duplicate styling properties, write border-radius: 0

Reply:
Wow sorry for this one 😆  Good catch 👍